### ISSUE TITLE
Improve submodel indentation

### ISSUE THREAD

**Issue by Alkly47** [xLightsSequencer/xLights#5072]
### **Is your feature request related to a problem? Please describe:**

Did the indentation in the sequencer change at some point? Or is it just now bothering me? Regardless, it is only a single character...
This makes it difficult to locate the top-most "parent" model/group when ready to collapse the expanded list.

### **Describe the solution you'd like:**

Add additional indentation, or add an option to increase it, so that it is easier to differentiate where the "parent" model/group starts vs layers, grouped models, or submodels in the sequencer. 

### **Describe alternatives you've considered:**

An icon, or hyphen, etc., preceding the submodel/layer or "parent" model/group, or even different colors would be helpful. Anything to help differentiate them from the top layer/group/model would be great. Options to tailor it to the user's preference would be even better!

### **Additional context:**

At first glance, differentiating the model vs submodel is difficult.... I'm hoping it can become more obvious.
![image](https://github.com/user-attachments/assets/8583bc05-31f3-471b-b4c4-417fdb93fcdd)

**Comment by cybercop23** [comment]
Yeah.. @derwin12 tried a variant. The grid can use an UX overhall.
